example-plugin/build.gradle

```diff
@@ -22,13 +22,6 @@ java {
 group 'com.yelp.nrtsearch.plugins'
 version '0.0.1'
 
-repositories {
-    mavenCentral()
```

Comment:
Should we still keep `mavenCentral()` here?

Reply:
There are two `repositories` blocks in this file for some reason https://github.com/Yelp/nrtsearch/pull/718/files/ab52631be40ff6bee2a75a7b58b93428d0547bfc#diff-ea04739ba1d9c1327164c174b91d0ed9ff951aee3753257514de672b9bf23b84R9